config/clusters/2i2c/utexas.values.yaml

```diff
@@ -89,6 +89,26 @@ jupyterhub:
           # So we put data in a subpath
           subPath: data
     initContainers:
+      # Repeat this here even though it's defined in basehub/values.yaml - otherwise
+      # tthe other initContainer we define here will overwrite this
```

Comment:
This addition is to 2i2c/utexas --- should it be centralized also?

Reply:
It needs to be repeated here, I left a comment explaining why there too - initContainers is a list so we can't just override it, as there is another initContainers here.

Reply:
👍👍👍